AntennaTracker/Tracker.h

```diff
@@ -79,7 +75,7 @@
 #include <SITL/SITL.h>
 #endif
 
-class Tracker : public AP_HAL::HAL::Callbacks {
+class Tracker : public AP_Vehicle, public AP_HAL::HAL::Callbacks {
```

Comment:
why multiple-inheritance? can't AP_Vehicle inherit from AP_HAL::HAL::Callbacks?

Reply:
Yes, it can.  The reasons for using multiple inheritance here are lost in the mists of caffeine.  I'll switch this around to have AP_Vehicle inherit from callbacks - and see what breaks.

Reply:
Changed to have `AP_Vehicle` inherit from callbacks.

One interesting aspect of doing things this way is that it forces Replay to implement those methods.  But that's probably worth it.